nodejs12.x/cookiecutter-quick-start-web/{{cookiecutter.project_name}}/package.json

```diff
@@ -4,10 +4,10 @@
     "version": "0.0.1",
     "private": true,
     "dependencies": {
-        "aws-sdk": "^2.437.0"
+        "aws-sdk": "^2.799.0"
```

Comment:
799 as software version 😱

Reply:
oh no it is 809 now... let's update it every once for a while